netcore/gen-xunit-runner/Program.cs

```diff
@@ -315,6 +318,43 @@ static int Main (string[] args)
 						}
 						if (!unhandled)
 							cases.Add (new TcCase () { Values = data });
+					} else if (cattr is MemberDataAttribute memberData) {
+						MethodInfo testDataMethod = m.DeclaringType.GetMethod (memberData.MemberName);
+						if (testDataMethod == null)
+							continue;
+
+						var rows = testDataMethod.Invoke (null, memberData.Parameters) as IEnumerable<object []>;
+						if (rows == null)
+							continue;
+
+						var notSupportedPrimitiveTypes = new [] {
+							typeof (IntPtr),
+							typeof (UIntPtr),
+							typeof (TimeSpan),
+							typeof (Guid),
+						};
+
+						foreach (ParameterInfo parameter in m.GetParameters ()) {
+							var pType = parameter.ParameterType;
+							// only primitive types + string are supported at this moment
+							if (pType.IsPrimitive && notSupportedPrimitiveTypes.Contains (pType))
+								goto notsupported;
+							
+							if (pType != typeof (string) && pType != typeof (decimal))
+								goto notsupported;
+						}
+
+						foreach (object[] data in rows) {
+							foreach (object dataItem in data) {
```

Comment:
This is supposed to be called at runtime. Not sure whenever it matters.

Reply:
yeah, this PR adds support only for simple cases (primitive literals, etc).